source/manual/taxonomy.html.md

```diff
@@ -98,6 +105,20 @@ a facet:
 
 [https://www.gov.uk/api/search.json?filter_part_of_taxonomy_tree=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&facet_taxons=1000&count=0](https://www.gov.uk/api/search.json?filter_part_of_taxonomy_tree=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&facet_taxons=1000&count=0)
 
+## Visibility
+
+Editors can use Whitehall to tag content to the taxonomy. They can see
+both the published as well as the draft taxonomy. Only content tagged
+to the published taxonomy can be used on the live site.
+
+Individual branches can be hidden from Editors by clearing the 
+`visible_to_departmental_editors` flag on level one taxons.
```

Comment:
Perhaps mention that this can be done through an interface in Content Tagger?

Reply:
fixed